src/store/creator/types.ts

```diff
@@ -6,6 +6,7 @@ export enum CreatorStatus {
     rejected = 'rejected',
     staged = 'staged',
     started = 'started',
+    reset = 'reset',
```

Comment:
Is this needed? It looks like the reducer sets status back to `init` when we cancel or the create annotation is fulfilled

Reply:
Not needed, removed it